drizzlepac/alignimages.py

```diff
@@ -918,46 +890,46 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, print_fit
         # if fit_rms_val < max_rms_val:
         #     fitRmsCheck = True
 
-        consistency_check = True
+        consistencyCheck = True
         rms_limit = max(item.meta['fit_info']['TOTAL_RMS'], 10.)
         if not math.sqrt(np.std(np.asarray(xshifts)) ** 2 + np.std(
-                np.asarray(yshifts)) ** 2) <= (rms_limit / 1000.0) / (item.wcs.pscale):
-            consistency_check = False
+                         np.asarray(yshifts)) ** 2) <= (rms_limit / 1000.0) / (item.wcs.pscale):
+            consistencyCheck = False
 
         # Decide if fit solutions are valid based on checks
-        if consistency_check is False:  # Failed consistency check
-            fit_status_dict[dict_key]['valid'] = False
-            fit_status_dict[dict_key]['compromised'] = False
-            fit_status_dict[dict_key]['reason'] = "Consistency violation!"
-        elif large_rms_check is False:  # RMS value(s) too large
-            fit_status_dict[dict_key]['valid'] = False
-            fit_status_dict[dict_key]['compromised'] = False
-            fit_status_dict[dict_key]['reason'] = "RMS too large (>150 mas)!"
-        elif radial_offset_check is False:  # Failed radial offset check
-            fit_status_dict[dict_key]['valid'] = False
-            fit_status_dict[dict_key]['compromised'] = False
-            fit_status_dict[dict_key]['reason'] = "Radial offset value too large!"
-        elif nmatches_check is False:  # Too few matches
-            fit_status_dict[dict_key]['valid'] = True
-            fit_status_dict[dict_key]['compromised'] = True
-            fit_status_dict[dict_key]['reason'] = "Too few matches!"
+        if consistencyCheck == False:  # Failed consistency check
```

Comment:
'==' should be 'is', or rewrite whole line as` if not consistencyCheck:`

Reply:
This syntax, and the same type of syntax used in this entire if...elif...else block will be changed to use "not <variable>".

Reply:
Resolved in 51e3cea